upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -3916,6 +3963,18 @@ spec:
               value: "true"
 
             # kops additions
+            # Enable source/destination checks for AWS
+            - name: FELIX_AWSSRCDSTCHECK
+              value: "{{- if and (eq .CloudProvider "aws") (.Networking.Calico.CrossSubnet) -}}Disable{{- else -}} {{- or .Networking.Calico.AwsSrcDstCheck "DoNothing" -}} {{- end -}}"
```

Comment:
You could consider using a options builder for calico to avoid this much logic in the template.

Reply:
Good idea, will think of doing it in a future PR.